src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    private String name;
```

Comment:
You actually don't use porperty `name` and `get / set / getMachineInfo` methods anywhere, so remove them